wasmbinding/stargate_whitelist.go

```diff
@@ -84,8 +85,6 @@ func init() {
 	setWhitelistedQuery("/osmosis.gamm.v1beta1.Query/CalcJoinPoolNoSwapShares", &gammtypes.QueryCalcJoinPoolNoSwapSharesResponse{})
 	setWhitelistedQuery("/osmosis.gamm.v1beta1.Query/PoolType", &gammtypes.QueryPoolTypeResponse{})
 	setWhitelistedQuery("/osmosis.gamm.v2.Query/SpotPrice", &gammv2types.QuerySpotPriceResponse{})
-	setWhitelistedQuery("/osmosis.gamm.v1beta1.Query/EstimateSwapExactAmountIn", &gammtypes.QuerySwapExactAmountInResponse{})
-	setWhitelistedQuery("/osmosis.gamm.v1beta1.Query/EstimateSwapExactAmountOut", &gammtypes.QuerySwapExactAmountOutResponse{})
```

Comment:
We still want to keep these to avoid breaking old contracts

Reply:
Brought them back for now, do we have further plans to deprecate them for the contracts though? I think it'd be awkward when in long term we support the two same queries but from different modules

Reply:
We would need to communicate with the integrators, asking whether anyone is using these. 

If contracts are already utilizing some of these, removal from the stargate-whitelist wouldn't be easy so we might have to accept keeping these while there are production users.

Reply:
sad 😭